content/en/profiler/profiler_troubleshooting/php.md

```diff
@@ -20,39 +20,48 @@ If you've configured the profiler and don't see profiles in the profile search p
 
 ## Reduce overhead from default setup
 
-If the default overhead is not acceptable, you can disable some of the sample
-types the profiler gathers by changing the following INI settings:
+If the default overhead is not acceptable, you can either [fine tune][4] the
+sampling distances, or disable some of the sample types the profiler gathers by
+changing the following INI settings:
 
 - `datadog.profiling.allocation_enabled`: controls allocation profiling
 - `datadog.profiling.experimental_cpu_time_enabled`: controls CPU-Time samples
 - `datadog.profiling.exception_enabled`: controls exception profiling
+- `datadog.profiling.timeline_enabled`: controls data gathering for the [timeline visualization][3]
 
 Disabling those sample types will leave you with only wall time samples being
 collected.
 
 See the [configuration docs][2] for other INI settings and their corresponding environment variables.
 
-## Exceptions overwhelming the profiler
+## Fine-tuning sampling distances
 
-The Datadog exception profiler has a small footprint and overhead under normal
-conditions. If a lot of exceptions are created and thrown, it can cause
-significant overhead for the profiler. This can happen when you use exceptions
-for control flow.
+Both the exception and allocation profilers are designed to have minimal impact
+under normal conditions. However, under high-load scenarios—such as frequent
+exception throwing or massive memory allocations—the sampling mechanism can
+contribute noticeable runtime cost.
 
-If you have an unusually high exception rate, you can either turn of exception
-profiling by setting `datadog.profiling.exception_enabled` to `0` or you can
-change the sampling distance via the
-`datadog.profiling.exception_sampling_distance` INI setting (default `100`) to a
-higher value. The higher the sampling distance, the fewer samples are created
-and the lower the overhead.
+For exceptions, this situation often arises when exceptions are used for
+control flow. For allocations, this is common when working with or processing
+large volumes of data. In both cases, the profiler may collect more samples
+than necessary, increasing its activity and resulting in higher overhead.
 
-See the [configuration docs][2] for the documentation on the exception sampling
-distance.
+To better manage this, you can adjust the respective sampling distances:
+
+- For exceptions, increase the sampling distance via the `datadog.profiling.exception_sampling_distance` INI setting (default: `100`). Alternatively, disable exception profiling entirely with `datadog.profiling.exception_enabled=0`.
+- For allocations, increase the sampling distance using `datadog.profiling.allocation_sampling_distance` (default: `4194304`, i.e., 4 MB). You can also disable allocation profiling with `datadog.profiling.allocation_enabled=0`.
```

Comment:
```suggestion
- For allocations, increase the sampling distance using `datadog.profiling.allocation_sampling_distance` (change the default `4194304` bytes, which is equivalent 4 MB, for example). You can also disable allocation profiling with `datadog.profiling.allocation_enabled=0`.
```

Reply:
The purpose of my suggestion is mainly to replace "i.e.", which is not ideal for translation or accessibility.